runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/metrics/Metric.java

```diff
@@ -32,12 +34,24 @@
         COUNT
     }
 
+    enum StreamDirection
+    {
+        RECEIVED,
+        SENT,
+        BOTH
+    }
+
     String name();
 
     Kind kind();
 
     Unit unit();
 
+    default StreamDirection streamDirection()
```

Comment:
Let's rename this method to `direction` too.

Reply:
done